packages/big-design/src/components/AccordionGroup/AccordionPanel/AccordionPanel.tsx

```diff
@@ -0,0 +1,55 @@
+import { Text } from '@bigcommerce/big-design';
+import { ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { HTMLAttributes, memo } from 'react';
+
+import { useUniqueId } from '../../../hooks';
+import { Box, BoxProps } from '../../Box/Box';
+import { useAccordion } from '../hooks/useAccordion';
+
+import { StyledAccordionButton, StyledAccordionContent } from './styled';
+
+export interface AccordionPanelProps extends HTMLAttributes<HTMLDivElement> {
+  children?: React.ReactNode;
+  defaultExpanded?: boolean;
+  header: string;
+  iconLeft?: React.ReactNode;
+  isExpanded: boolean;
+  onClick?: () => void;
+}
+
+export const AccordionPanel: React.FC<AccordionPanelProps & BoxProps> = memo(
+  ({ children, header, iconLeft, defaultExpanded = false, className, style, ...props }) => {
+    const { isExpanded, onClick } = useAccordion(defaultExpanded);
+    const accordionId = useUniqueId('accordion');
+    const accordionItemId = useUniqueId('accordion-item');
+
+    return (
+      <Box {...props}>
+        <StyledAccordionButton
+          aria-controls={accordionItemId}
+          aria-expanded={isExpanded}
+          iconLeft={iconLeft}
+          iconRight={<ExpandMoreIcon className="collapse-icon" color="secondary70" />}
+          id={accordionId}
+          isExpanded={isExpanded}
+          onClick={onClick}
+          variant="subtle"
+        >
+          <Text marginBottom="none" marginLeft={iconLeft ? 'xxxLarge' : 'none'}>
```

Comment:
Since this is using the `StylableButton`, couldn't we just apply some grid CSS to the `span` instead of passing a `p` tag into a `button`? Trying to see if we can cut unnecessary html elements from the DOM.